src/seedu/addressbook/parser/Parser.java

```diff
@@ -85,7 +85,7 @@ public Command parseCommand(String userInput) {
             return new ClearCommand();
 
         case FindCommand.COMMAND_WORD:
-            return prepareFind(arguments);
+            return prepareFind(arguments.toLowerCase());
```

Comment:
Here you only converted the keywords to lower case. Will this still work if the names in the address are not all lower-case?